pkg/apiserver/archive/archive.go

```diff
@@ -51,6 +51,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 	endpoint.GET("", s.list)
 	endpoint.GET("/detail", s.detail)
 	endpoint.GET("/report", s.report)
+	endpoint.DELETE("/:uid", s.delete)
```

Comment:
```suggestion
	endpoint.DELETE("/uid", s.delete)
```

Reply:
A ':' is needed because it means that this is a parameter called "uid".

Reply:
ok